src/components/FeatureWalkthrough/FeatureWalkthrough.tsx

```diff
@@ -0,0 +1,203 @@
+import { FunctionComponent, useEffect } from 'react'
+
+import classNames from 'classnames'
+import { startCase } from 'lodash'
+import Link from 'next/link'
+
+import { buttonStyle, buttonLocation } from '@data'
+
+import batchChangesMp4 from '../../pages/home/assets/animations/batch-changes.mp4'
+import batchChangesWebm from '../../pages/home/assets/animations/batch-changes.webm'
+import codeInsightsMp4 from '../../pages/home/assets/animations/code-insights.mp4'
+import codeInsightsWebm from '../../pages/home/assets/animations/code-insights.webm'
+import codeIntelMp4 from '../../pages/home/assets/animations/code-intel.mp4'
+import codeIntelWebm from '../../pages/home/assets/animations/code-intel.webm'
+import codeSearchMp4 from '../../pages/home/assets/animations/code-search.mp4'
+import codeSearchWebm from '../../pages/home/assets/animations/code-search.webm'
```

Comment:
I would move the assets to a collocated `animations` directory since they should be scoped to this component and not the homepage. The homepage inherits the FeatureWalkthrough component. 

```suggestion
import batchChangesMp4 from './animations/batch-changes.mp4'
import batchChangesWebm from './animations/batch-changes.webm'
import codeInsightsMp4 from './animations/code-insights.mp4'
import codeInsightsWebm from './animations/code-insights.webm'
import codeIntelMp4 from './animations/code-intel.mp4'
import codeIntelWebm from './animations/code-intel.webm'
import codeSearchMp4 from './animations/code-search.mp4'
import codeSearchWebm from './animations/code-search.webm'
```